### PR TITLE
9 Fix pagination links

### LINKED ISSUE #9: Next/previous page links can be wrong in the presence of filtering
When we expect

http://127.0.0.1:8005/path/models/?thing=1&page=2

The link to the next/last page is 

http://127.0.0.1:8005/path/models/?thing=%5Bu%271%27%5D&page=2

which decodes to `thing=[u'1']`.

### PR DESCRIPTION
**Pull request by dpoirier**
Fixes #9

### CODE REVIEW

**Comment by dpoirier** [comment]
Don't review yet, it looks like `doseq=True` on urlencode is a more correct fix.

**Comment by dpoirier** [comment]
Updated with better fix.

File: bread/bread.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def get_form(self, data=None, files=None, **kwargs):
             self.form_class = modelform_factory(
                 self.bread.model,
                 fields='__all__',
-                exclude=self.exclude
+                exclude=self.bread.exclude
```
Comment (trawick) [inline bread/bread.py]: What does this fix/how to test?

Comment (dpoirier) [inline bread/bread.py]: This fixes a test failure (run tests by running "tox").  Philip also ran into it when he started using bread for some new views. He included this fix in #15 anyway, so this change is already in develop.

**Comment by vkurup** [comment]
:+1:

**Pull request merged**